k8s/docker-entrypoint.sh

```diff
@@ -48,7 +48,7 @@ restore() {
 
 grpc() {
     echo "Starting Medusa gRPC service"
-    python3 -m medusa.service.grpc.server server.py
+    exec python3 -m medusa.service.grpc.server server.py
```

Comment:
Out of curiosity, what's the difference with using `exec` instead of calling `python3` directly? Is it related to signal handling?

Reply:
`exec` causes the new process to be run with in the current shell/process.

Here is the `ps` output from within a `medusa` container that does **not** use `exec`:

```
$ ps -x
    PID TTY      STAT   TIME COMMAND
      1 ?        Ss     0:00 /bin/bash /home/cassandra/docker-entrypoint.sh
      7 ?        Sl     0:00 python3 -m medusa.service.grpc.server server.py
    372 pts/0    Ss     0:00 /bin/bash
```
Running `kill -SIGTERM 1` has no effect here. 

Here is the `ps` output from a `medusa` container where `exec` is used:

```
$ ps -x
    PID TTY      STAT   TIME COMMAND
      1 ?        Ssl    0:00 python3 -m medusa.service.grpc.server server.py
    347 pts/0    Ss     0:00 /bin/bash
```
Running `kill -SIGTERM 1` has the desired effect here.

Reply:
kk, thanks